src/components/organization/organization.service.ts

```diff
@@ -220,12 +220,12 @@ export class OrganizationService {
             count(org) as orgCount
           `,
           {
-            token: session.token,
+            token: session?.token,
```

Comment:
I think `checkAllOrgs` should just require `session`. The db query doesn't make sense without it.

Reply:
Makes perfect sense, updating now.